env/CMakeLists.txt

```diff
@@ -67,4 +68,19 @@ ExternalProject_Add(
     DEPENDS python-venv
 )
 
+ExternalProject_Add(
+    onnx-mlir
+    PREFIX ${TTMLIR_TOOLCHAIN_DIR}
+    CMAKE_GENERATOR Ninja
+    CMAKE_ARGS
+    -DCMAKE_CXX_COMPILER=/usr/bin/c++
+    -DPython3_ROOT_DIR=$pythonLocation
+    -DMLIR_DIR=${TTMLIR_TOOLCHAIN_DIR}/llvm-project/build/lib/cmake/mlir
+    -DCMAKE_INSTALL_PREFIX=${TTMLIR_TOOLCHAIN_DIR}
```

Comment:
Can we also add `-DONNX_MLIR_BUILD_TESTS=OFF -DONNX_MLIR_ENABLE_JAVA=OFF`?

Reply:
Alright

Reply:
Since build tests are disable, so package onnx is not required anymore, so I removed it from dependencies.